src/styles/blocks/container.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  --columm-count: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--columm-count), 200px);
```

Comment:
Ensure that the custom property `--columm-count` is corrected to `--column-count` here as well to maintain consistency and avoid errors.